### PR TITLE
NO-TICKET-Alignment-Ingredient

### PR DESCRIPTION
**Pull request by thelrc**
Made changes to custom-searchbar.js CSS to align both the search bar with the ingredient search

### CODE REVIEW

**Review by ahthai2468 — APPROVED** [review]
nice, I think this looks better :)

**Review by gmihir — CHANGES_REQUESTED** [review]
looks great on desktop! i think the mobile formatting needs to be addressed, but overall this looks great

image

**Comment by thelrc** [comment]
Update: I fixed the issue with adding flex box but introduced more issues that I do not know how to fix.

Issue 1 - For some reason I can not align the width of `.bar  and .collapsible`. When the window is minimized to smaller screens is when the issue happens. The issue might be having padding and margins elsewhere but don't know where?  

Issue 2 - After switching to flex box. I can't find a way to move the +/- sign in Search by Ingredient button since its only one element.

Issue 3 - Also alignment issues with inputs ingredients-included and ingredients-excluded

File: root/components/custom-searchbar.js [hunk 1]
```diff
@@ -221,17 +217,17 @@ class Searchbar extends HTMLElement {
...
               font-family: 'Nunito', sans-serif !important;
-              margin-top: calc(1vh);
+              height: 50px;
+              margin: calc(1vh) auto;
               background-color: #eee;
               color: #444;
               cursor: pointer;
-              padding: 18px;
-              width: 80%;
+              width: 81%;
```
Comment (thelrc) [inline root/components/custom-searchbar.js:227]: Issue 1

File: root/components/custom-searchbar.js [hunk 2]
```diff
@@ -155,9 +155,11 @@ class Searchbar extends HTMLElement {
               }
             }
             .bar{
+                display: flex;
                 margin: 0 auto;
+                padding: 5px;
                 width: 80%;
```
Comment (thelrc) [inline root/components/custom-searchbar.js:161]: Issue 1

**Pull request closed without merging**